docs/releases/3.1.txt

```diff
@@ -51,6 +51,11 @@ Minor features
   instead of deprecated ``PASSWORD_RESET_TIMEOUT_DAYS``, which will be removed
   in Django 4.0.
 
+* Added the :setting:`TEST_PASSWORD_HASHER` setting to define a faster
+  password algorithm when running tests. It uses an basic SHA1-based algorithm
+  by default. It's possible to opt-out from this behavior by setting its value
+  to ``None``.
```

Comment:
Shouldn't this be under `Tests`?

Reply:
Good question. Not sure which section is best, but I followed your suggestion, and move it under tests.